packages/create-cloudflare/templates/hello-world/js/test/index.spec.js

```diff
@@ -0,0 +1,18 @@
+import { env, createExecutionContext, waitOnExecutionContext, SELF } from "cloudflare:test";
+import { describe, it, expect } from "vitest";
+import worker from "../src";
+
+describe("Hello World worker", () => {
+  it("displays Hello World! (unit style)", async () => {
+    const request = new Request("http://example.com");
+    const ctx = createExecutionContext();
+    const response = await worker.fetch(request, env, ctx);
+    await waitOnExecutionContext(ctx);
```

Comment:
Given that this is as much a learning tool as a kickstarter, would it be useful to add some comments, such as explaining what `waitOnExecutionContext()` does and why it is needed here?